presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -54,6 +55,8 @@
     private Duration queryMaxRunTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
 
+    private int minWorkerNodesInitialized = 1;
```

Comment:
We should find a better name for this config and related variables. How about `initialNumberOfWorkersRequired`?